example/android/app/src/debug/java/com/spotlightexample/ReactNativeFlipper.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Branding update! haha

Reply:
Yeah 😂 this comes from the [React Native Upgrade Helper tool](https://react-native-community.github.io/upgrade-helper/?from=0.66.4&to=0.69.3), which will make future updates easier too 😁